src/Sentry/DynamicSamplingContext.cs

```diff
@@ -127,6 +127,14 @@ public static DynamicSamplingContext CreateFromTransaction(TransactionTracer tra
             userSegment,
             transactionName);
     }
+
+    public static DynamicSamplingContext CreateFromPropagationContext(SentryPropagationContext propagationContext, SentryOptions options)
+    {
+        var publicKey = Dsn.Parse(options.Dsn!).PublicKey;
+        var traceId = propagationContext.TraceId;
+
+        return new DynamicSamplingContext(traceId, publicKey, 1.0f);
+    }
```

Comment:
@bitsandfoxes I think you'll want to propagate/set the [new `sampled` property](https://github.com/getsentry/sentry-dotnet/blob/55d4f22e81861a2e0982cdf96a79c4135e8f03e1/src/Sentry/DynamicSamplingContext.cs#L25) in the DSC as well.

Reply:
We do. With this change, all errors and transactions sent should always contain a DSC.